examples/computer_vision/efficientdet_pytorch/model_def.py

```diff
@@ -0,0 +1,409 @@
+from typing import Any, Dict, Sequence, Tuple, Union, cast
+import yaml
+import logging
+from copy import deepcopy
+
+import torch
+from apex import amp
+from determined import pytorch
+import math
+import numpy as np
+
+from horovod.torch.sync_batch_norm import SyncBatchNorm
+
+
+from pycocotools.cocoeval import COCOeval
+from efficientdet_files.evaluator import *
+from efficientdet_files.modelema import ModelEma
+
+
+from effdet import create_model, create_loader, create_dataset
+from effdet.data.loader import DetectionFastCollate
+from effdet.data import resolve_input_config, SkipSubset
+from effdet.data.transforms import *
+from effdet.anchors import Anchors, AnchorLabeler
+
+from timm.optim import create_optimizer
+from timm.scheduler import create_scheduler
+
+from determined.pytorch import DataLoader, PyTorchTrial, PyTorchTrialContext, LRScheduler
+from determined.experimental import Determined
+
+TorchData = Union[Dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor]
+
+MAX_NUM_INSTANCES = 100
+
+
+class EffDetTrial(PyTorchTrial):
+    def __init__(self, context: PyTorchTrialContext) -> None:
+        self.context = context
+        self.hparam = self.context.get_hparam
+        self.args = DotDict(self.context.get_hparams())
```

Comment:
You define class `DotDict` at the bottom of the model_def.py. Could you maybe move it to the top that way it's just easy to recall this usage here?